docs/changelog-fragments.d/459.feature.md

```diff
@@ -0,0 +1,9 @@
+Added the ability to run builder commands to create execution
+environment using mode `stdout`.
```

Comment:
(important) Let's try to use the full names of things in the changelog. This is because (1) there's a generic Ansible docs writing style guide emphasizing this and (2) it is useful for the consumers of the changelog to have enough context into what the entry is about that they wouldn't need to do extra googling/code reading unless they actually want to dig deep.
```suggestion
Added the ability to run ansible-builder commands to create
execution environment using mode `stdout`.
```

Reply:
Fixed